packages/cli/src/api/workflows.api.ts

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable no-param-reassign */
+/* eslint-disable import/no-cycle */
+
+import express from 'express';
+import { IDataObject, INode, LoggerProxy } from 'n8n-workflow';
+
+import { Db, ResponseHelper, whereClause, WorkflowHelpers } from '..';
+import config from '../../config';
+import * as TagHelpers from '../TagHelpers';
+import { SharedWorkflow } from '../databases/entities/SharedWorkflow';
+import { WorkflowEntity } from '../databases/entities/WorkflowEntity';
+import { validateEntity } from '../GenericHelpers';
+import { InternalHooksManager } from '../InternalHooksManager';
+import { externalHooks } from '../Server';
+import type { WorkflowRequest } from '../requests';
+
+export const workflowsController = express.Router();
```

Comment:
Diff obscured by moving files - only changes to endpoints are at the top of POST and at the bottom of GET.